gamepanel/game.php

```diff
@@ -143,12 +143,16 @@ function gameTimeRemaining()
 		$timerCount++;
 
 		if( $this->phase == 'Pre-game' )
-			$buf = '<span class="gameTimeRemainingNextPhase">'.l_t('Start:').'</span> '.
-				$this->processTimetxt().' ('.libTime::detailedText($this->processTime).')';
+			$buf =
+				'
+					<span class="gameTimeRemainingNextPhase">'.l_t('Start:').'</span> '.$this->processTimetxt().' <span class="timestampGamesWrapper"> ('.libTime::detailedText($this->processTime).') </span>
+				';
 		else
 		{
-			$buf = '<span class="gameTimeRemainingNextPhase">'.l_t('Next:').'</span> '.
-				$this->processTimetxt().' ('.libTime::detailedText($this->processTime).')';
+			$buf =
+				'
+					<span class="gameTimeRemainingNextPhase">'.l_t('Next:').'</span> '.$this->processTimetxt().' <span class="timestampGamesWrapper"> ('.libTime::detailedText($this->processTime).') </span>
+				';

```

Comment:
it looks like 'Next' and 'Start' are the only things changing here. can we simplify this logic to have a ternary for the text that goes in the span?